File: overview/rev1/guides/examples/java/card-credit.java
// cardHref is the stored href for the card
Card card = new Card(cardHref);

HashMap<String, Object> payload = new HashMap<String, Object>();
payload.put("amount", 5000);
payload.put("description", "Some descriptive text for the debit in the dashboard");
payload.put("appears_on_statement_as", "Some text");

try {
    Credit credit = card.credit(payload);
}
catch (FundingInstrumentNotCreditable e) {}
catch (HTTPError e) {}

File: overview/rev1/guides/examples/csharp/card-credit.cs
# card_href is the stored href for the Card
Card card = Card.Fetch(card_href);
Dictionary<string, object> creditPayload = new Dictionary<string, object>();
creditPayload.Add("amount", 5000);
creditPayload.Add("description", "Some descriptive text for the debit in the dashboard");
Credit credit = card.Credit(creditPayload);